src/pages/EnablePayments/OnfidoStep.js

```diff
@@ -0,0 +1,154 @@
+
+        /** Whether the user has accepted the privacy policy of Onfido or not */
+        hasAcceptedPrivacyPolicy: PropTypes.bool,
+    }),
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    walletOnfidoData: {
+        loading: false,
+        hasAcceptedPrivacyPolicy: false,
+    },
+};
+
+class OnfidoStep extends React.Component {
+    canShowOnfido() {
+        return this.props.walletOnfidoData.hasAcceptedPrivacyPolicy
+            && !this.props.walletOnfidoData.loading
+            && !this.props.walletOnfidoData.error
+            && this.props.walletOnfidoData.sdkToken;
+    }
+
+    render() {
+        return (
+            <>
+                <HeaderWithCloseButton
+                    title={this.props.translate('onfidoStep.verifyIdentity')}
+                    onCloseButtonPress={() => Navigation.goBack()}
+                />
+                {
+                    this.canShowOnfido() ? (
+                        <Onfido
+                            sdkToken={this.props.walletOnfidoData.sdkToken}
+                            onUserExit={() => {
+                                Navigation.goBack();
+                            }}
+                            onSuccess={(data) => {
+                                activateWallet(CONST.WALLET.STEP.ONFIDO, {
+                                    onfidoData: JSON.stringify({
+                                        ...data,
+                                        applicantID: this.props.walletOnfidoData.applicantID,
+                                    }),
+                                });
+                            }}
+                        />
+                    ) : (
+                        <View style={[styles.mh5, styles.mb5, styles.flex1, styles.justifyContentBetween]}>
+                            {!this.props.walletOnfidoData.hasAcceptedPrivacyPolicy && (
+                                <>
+                                    <View style={styles.justifyContentCenter}>
+                                        <Text style={[styles.mb5]}>
+                                            {this.props.translate('onfidoStep.acceptTerms')}
+                                            <TextLink
+                                                href="https://onfido.com/facial-scan-policy-and-release/"
+                                            >
+                                                {this.props.translate('onfidoStep.facialScan')}
+                                            </TextLink>
+                                            {', '}
+                                            <TextLink
+                                                href="https://onfido.com/privacy/"
+                                            >
+                                                {this.props.translate('common.privacyPolicy')}
+                                            </TextLink>
+                                            {' and '}
```

Comment:
I think we should use a translation here too

Reply:
Oh, nice catch. I agree.